src/Components/Shared/Pagination.css

```diff
@@ -0,0 +1,34 @@
+.pagination {
+    border-radius: 0.25rem;
+    display: inline-block;
+    margin: 1.25rem 0;
+    padding-left: 0;
+}
+
+.pagination .page-item {
+    display: inline-block;
+}
+
+.pagination li button {
+    background: #fff;
+    border: 1px solid #e6e6e6;
+    border-radius: 0.125rem;
+    color: #000;
+    line-height: 1.42857;
+    margin: 0.125rem;
+    padding: 0.625rem 1.25rem;
+    text-decoration: none;
+}
+
+.pagination li.active button.page-link {
+    background: #4aca85;
```

Comment:
Remember we talked about to use css variables, we can use it for colors for start.